src/components/molecules/Inbox.vue

```diff
@@ -12,17 +12,22 @@
   </span>
   <!-- Loaded State -->
   <span v-else>
+    <span :id="'inboxHeaderDesc'" class="hidden">
+      Inbox header, press tab or click to access inbox items
```

Comment:
This doesn't work as expected when clicking enter here with NVDA

Reply:
This might've been a misunderstanding, it's supposed to mean that you tab to or click on the inbox items. If you can't access the label, try using the skip nav as the label can only be seen when you focus on it. 

As a side note, I'd also recommend looking into NVDA's Browse mode, it allows you to focus on most elements by going to the closest element of X type by pressing a letter e.g. pressing 'h' or shift+'h' lets you go to next/previous heading, and each number lets you go to a heading of that level e.g. pressing '1' makes you go to the closest h1